lib/datadog/core/configuration/ext.rb

```diff
@@ -19,6 +19,11 @@ module Metrics
 
         module Transport
           ENV_DEFAULT_HOST = 'DD_AGENT_HOST'
+          # DD_TRACE_AGENT_PORT is the only env var available for setting the agent port.
+          # It is the same as Datadog::Tracing::Configuration::Ext::Transport::ENV_DEFAULT_PORT
+          # but is also here as we do not want Core to have a dependency on Tracing.
+          ENV_DEFAULT_PORT = 'DD_TRACE_AGENT_PORT'
```

Comment:
I think the `_TRACE_` in `DD_TRACE_AGENT_PORT` is a (legacy?) misnomers, given this port is the communication port for all of APM products with the agent. It's documented as `APM Receiver`: https://github.com/DataDog/documentation/blob/318dffe3a8c86aa81fc3b6619c9f19ab4e48af41/content/en/agent/guide/network.md?plain=1#L354-L355

I'd read this environment variable as `DD_APM_AGENT_PORT`.

I don't want to throw a wrench at this specific change, just want to bring up the fact that this this port is not tracing-specific.

Reply:
So this change is just copying the consts for the existing env vars into the core namespace so we can reference it in core without including any tracing code and maintain backwards compatibility. I agree the name is probably not accurate since the port isn't tracing specific anymore. From conversations i've had, changing the actual env var to remove the `_tracing_` part would require effort beyond the scope of this project.